corehq/apps/app_manager/static/app_manager/template_apps/health/multimedia.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This replaces [appcues/multimedia.json](https://github.com/dimagi/commcare-hq/blob/8fdbf910431a19bcbc24dd221d1d000c5f25829b/corehq/apps/app_manager/static/app_manager/template_apps/appcues/multimedia.json), right? Can you delete that file? Same for `app.json`.

Reply:
Yeah, it does. I can get rid of the `appcues` folder.
Do you think it's safe for me replace the `appcues` folder with `health`? Or is do we need to keep the old appcues folder around for anything/anyone who might be using the old `app_from_template/appcues` url and expecting a health-related app?

Reply:
It's safe to replace, that's not a url that users should know about or be using directly.